python/tskit/trees.py

```diff
@@ -171,7 +173,8 @@ class Edge(SimpleContainer):
         :meth:`.TreeSequence.node`.
     :vartype child: int
     """
-    def __init__(self, left, right, parent, child):
+    def __init__(self, id_, left, right, parent, child):
```

Comment:
Put ``id_=None`` at the end here. This'll break less code.

Reply:
OK. Note that in the tests these constructors are often called using positional params, so this will make it inconsistent with some of the `Node()` etc examples. But better that than breaking other people's code, I guess.

Presumably best to change the calls to use named parameters in the test cases too.

Reply:
Yes, that's fine --- these things aren't particularly consistent anyway. There's no need to change existing tests if they're not actually using the ID parameter.

Reply:
Done